packages/hoppscotch-common/src/services/persistence/validation-schemas/index.ts

```diff
@@ -276,7 +276,8 @@ const HoppGQLSaveContextSchema = z.nullable(
       .object({
         originLocation: z.literal("user-collection"),
         folderPath: z.string(),
-        requestIndex: z.number(),
+        // TODO: Investigate why this field is not populated at times
+        requestIndex: z.optional(z.number()),
```

Comment:
Encountered an issue while creating GraphQL requests where entries under the tree and tab are not associated. The schema update is made not to show the error toast anymore - to be reverted when the behavior is resolved. Tracked in [HFE-546](https://linear.app/hoppscotch/issue/HFE-546/request-from-the-graphql-collection-tree-is-not-associated-with-the).